ERROR    root:test_httpRun.py:152 assert 400 == 200
 +  where 400 = <Response [400]>.status_code
INFO     root:logDecoratorl.py:39 
>>===============================正在执行第6条用例======================================>>
测试标题: 密码为空，登录失败
请求方式: post
请求头:   {'accesstoken': '17b471073252f2cc3790c054591aaf18', 'timespan': '1660615538'}
请求路径: http://110.80.137.11:8051/Users/Login_UserName
传参方式： params
请求内容: {
                "phone": "",
                "username":13800000008,
                "password": "",
                "code": "",
                "invitecode": ""
            }
依赖测试用例: 暂无依赖用例数据
接口响应内容: {"type":"https://tools.ietf.org/html/rfc7231#section-6.5.1","title":"One or more validation errors occurred.","status":400,"traceId":"00-f42435d34fb2a9da52887649f8c84494-5799c745e8d44d63-00","errors":{"password":["The password field is required."]}}
接口响应时长: 0.081395 s
Http状态码: 400
断言方式：相等
期望结果：False
实际结果：{'type': 'https://tools.ietf.org/html/rfc7231#section-6.5.1', 'title': 'One or more validation errors occurred.', 'status': 400, 'traceId': '00-f42435d34fb2a9da52887649f8c84494-5799c745e8d44d63-00', 'errors': {'password': ['The password field is required.']}}
>>==============================第6条用例执行完毕=======================================>>